File: d2c/WebContent/WEB-INF/services/transferable_course_instance.js
app.factory('transferable_course_instance',[function(){
	return function(year_offered, semester, crn, course){
		return {
			'semester': semester,
			'year_offered': year,
			'profName': profName,
			'crn': courseReferenceNumber,
			'courseID': courseID,
			'refID':refID
		};
	};
}]);

File: d2c/src/com/d2c/web/beans/TransferableCourseInstance.java
package com.d2c.web.beans;

//import com.d2c.util.Participatable;

public class TransferableCourseInstance {
	public Semester semester; //WINTER
	public String year; //2016
	public String profName;
	public String courseReferenceNumber; //00001
	public int courseID;
	public int refID;
	
	public enum Semester {
		FALL,
		WINTER,
		SPRING
	}
}

